linter_exclusions.yml

```diff
@@ -8,6 +8,16 @@ iot ops asset query:
     instance_resource_group:
```

Comment:
Quite a bit of exclusions just for parameter_should_not_end_in_resource_group, I guess a wild idea is to call the param something else but similar :smile: